R/license.R

```diff
@@ -11,7 +11,11 @@
 #'   provides patent protection.
 #' * [GPL v3](https://choosealicense.com/licenses/gpl-3.0/): requires sharing
 #'   of improvements.
-#' * [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/): Free to share and
+#' * [AGPL v3](https://choosealicense.com/licenses/gpl-3.0/): requires sharing
```

Comment:
This URL looks wrong. Shouldn't it be:

https://choosealicense.com/licenses/agpl-3.0/